docs/product-user-guide/src/main/asciidoc/guided-decision-tables-columns-action-BRL-con.adoc

```diff
@@ -1,13 +1,19 @@
 [id='guided-decision-tables-columns-action-BRL-con']
 = "Add an Action BRL fragment"
 
-A Business Rule Language (BRL) fragment is a section of a rule created using the Guided Rule Editor. The <<guided-decision-tables-columns-condition-BRL-con, Condition BRL fragment>> is the "WHEN" portion of the rule, and the Action BRL fragment is the "THEN" portion of the rule. With this column option you can define an Action BRL fragment to be used in the right ("THEN") side of a rule. When using the embedded Guided Rule Editor, field values defined as "Template Keys" form columns in the decision table.
-Simpler column types can refer to Facts and Fact fields bound in the BRL fragment and vice-versa.
+A Business Rule Language (BRL) fragment is a section of a rule created using the guided rule designer. The xref:guided-decision-tables-columns-condition-BRL-con[condition BRL fragment] is the "WHEN" portion of the rule, and the action BRL fragment is the "THEN" portion of the rule. With this column option you can define an action BRL fragment to be used in the right ("THEN") side of a rule. Any *Template key* field values (as opposed to *Literal* or *Formula* values) that you add to conditions (example: `$rate`) form separate columns in the decision table. Simpler column types can refer to Facts and Fact fields bound in the BRL fragment and vice-versa.
 
 Example action BRL fragment for a loan application:
 
-.Simple action BRL fragment
-image::guided-decision-tables-columns-action-BRL.png[A Simple Action BRL Fragment in the guided decision table editor for BRMS User Guide 6.0.2]
+.Add an action BRL fragment with the embedded guided rule designer
+image::guided-decision-tables-columns-action-BRL.png[Action BRL Fragment in the guided decision table designer]
+
+You can also select *Add free form DRL* from the list of action options to define the action BRL fragment without the embedded guided rule designer.
```

Comment:
Capitalise "Free Form DRL"?

Reply:
Not capped in UI. But wouldn't cap anyway. See my overall comment in your review comment. Won't mention others, just noting here.